test/test-k8s.bats

```
@@ -201,6 +222,11 @@ teardown_file() {
 }
 
 @test "kube-burner init: jobType kubevirt" {
+  # Only skip if explicitly told to do so by SKIP_KUBEVIRT_TESTS
+  # This ensures we catch real failures in CI instead of silently skipping
+  if [[ "${SKIP_KUBEVIRT_TESTS:-false}" == "true" ]]; then
```

I'm not sure if this is still relevant, but tests mustn't be skipped due to installation failure. Only of requested. If this is the case, just fix the comment

---

Added a comprehensive "ERROR HANDLING POLICY" section at the top of [helpers.bash](https://bug-free-meme-pjp7x9769rrg36wjp.github.dev/) that explicitly states:

All setup failures result in explicit test failure (exit 1), NEVER silent skipping
Tests are ONLY skipped when user explicitly sets SKIP_* variables
All errors use "FATAL:" prefix for consistency and easy identification
No automatic retry logic or fallbacks - we fail fast and explicitly.

Enhanced the comments around the SKIP_KUBEVIRT_INSTALL section to be more explicit:

Changed the comment to include "ONLY skips if user explicitly requests it"
Added a NOTE that clarifies "We only set SKIP_*_TESTS variables when user explicitly requests it, never due to failures"
Made it clear that CDI skip is also based on user request
The changes make it absolutely clear that:

No automatic skipping occurs
SKIP_* variables are only set based on user-specified input
All failures result in explicit test failure, not silent skipping.